src/transformState.js

```diff
@@ -5,7 +5,40 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
+    switch ({
+      type,
+      extraData,
+      keysToRemove,
+    }.type) {
```

Comment:
sorry, don't get it😅 why just do not do this?
```suggestion
    switch (type) {
```